envs/feedstock-patches/numba/0001-update-to-numba-0.56.1-and-support-numpy-1.23.patch

```diff
@@ -1,14 +1,34 @@
-From 7f154fd52e039f0047576f066504952b1e228d19 Mon Sep 17 00:00:00 2001
+From c0e16167951d45fb7d6fcde4e0256c50f88dcd1c Mon Sep 17 00:00:00 2001
 From: Deepali Chourasia <deepch23@in.ibm.com>
-Date: Thu, 6 Oct 2022 16:19:16 +0000
+Date: Fri, 7 Oct 2022 06:43:08 +0000
 Subject: [PATCH] update to numba 0.56.1 and support numpy 1.23
 
 ---
- recipe/meta.yaml | 22 ++++++++++------------
- 1 file changed, 10 insertions(+), 12 deletions(-)
+ recipe/0001-unpin-setuptools.patch | 12 ++++++++++++
+ recipe/meta.yaml                   | 22 +++++++++++-----------
+ 2 files changed, 23 insertions(+), 11 deletions(-)
+ create mode 100644 recipe/0001-unpin-setuptools.patch
 
+diff --git a/recipe/0001-unpin-setuptools.patch b/recipe/0001-unpin-setuptools.patch
+new file mode 100644
+index 0000000..bb6c9e3
+--- /dev/null
++++ b/recipe/0001-unpin-setuptools.patch
+@@ -0,0 +1,12 @@
++diff --git a/setup.py b/setup.py
++index 2a6d12277..20948975f 100644
++--- a/setup.py
+++++ b/setup.py
++@@ -375,7 +375,6 @@ build_requires = ['numpy >={},<{}'.format(min_numpy_build_version,
++ install_requires = [
++     'llvmlite >={},<{}'.format(min_llvmlite_version, max_llvmlite_version),
++     'numpy >={},<{}'.format(min_numpy_run_version, max_numpy_run_version),
++-    'setuptools<60',
```

Comment:
Will numba work with setuptools 63.* when they have mentioned this hard requirement in setup.py?

Reply:
Builds and import works. I am going to build snapml and try numba tests too.

Reply:
As we are already using setuptools 63.* in OpencE 1.7.2 this is already tested. Just that pip check was failing. So this change will avoid the pip check failure.

Reply:
Okay. Great!